### PR TITLE
CO-1755 implement identities API v2

### PR DESCRIPTION
**Pull request by tsoliangwu0130**

### CODE REVIEW

File: src/api-routes/identities.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+import { errorBuilder, errorHandler } from 'errors/errors';
+import { getIdentities } from '../db/oracledb/identities-dao';
+
+/**
+ * Get pets
```
Comment (Ruefa) [inline src/api-routes/identities.js]: Update JSdoc

File: src/api-routes/identities/{osuId}.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { errorBuilder, errorHandler } from 'errors/errors';
+import { getIdentityById } from '../../db/oracledb/identities-dao';
+
+/**
+ * Get pet by unique ID
```
Comment (Ruefa) [inline src/api-routes/identities/{osuId}.js]: JSdoc

File: src/db/oracledb/identities-dao.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import { getConnection } from './connection';
+import { contrib } from './contrib/contrib';
+
+/**
+ * Return parsed query with keys changed to remove the "filter[]" wrapper
+ *
+ * @param {object} query query parameters
+ * @returns {object} parsed query
+ */
+const parseQuery = (query) => {
```
Comment (Ruefa) [inline src/db/oracledb/identities-dao.js]: Should this go in `utils` in the skeleton? I imagine other APIs could use this including some persons endpoints I haven't gotten to yet.

**Review by Ruefa — CHANGES_REQUESTED** [review]

File: src/db/oracledb/identities-dao.js (hunk 3 above)
Comment (tsoliangwu0130) [reply src/db/oracledb/identities-dao.js]: Good idea, I'll create a PR in skeleton.

**Review by Ruefa — APPROVED** [review]

**Pull request merged**